src/components/Placeholders/FeaturedPlaceholdersList.jsx

```diff
@@ -47,10 +49,10 @@ const FeaturedPlaceholdersList = ({ featuredPlaceholders }) => {
         {featuredPlaceholders.map((placeholder, idx) => (
           <Placeholder
             key={idx}
-            ref={actionBtnRef}
+            ref={el => (actionBtnRefs.current[idx] = el)}
             placeholder={placeholder}
             divider={idx !== featuredPlaceholders.length - 1}
-            onClick={showImportDropdown}
+            onClick={showImportDropdown(idx)}
```

Comment:
if it was a regression, we need to add a test to prevent this regression to happen again 🙏

Reply:
@trollepierre J'aimerai bien, mais je ne vois pas comment tester rapidement l'ancrage d'un actionMenu sur un div. Si tu as une idée je suis preneur, sinon je propose de faire une issue.